### ISSUE TITLE
Wrong value for Height and Width on Flutter build desktop

### ISSUE THREAD

**Issue by fahmi277** [TechnoPrashant/Sizer#27]
Hello dev,

I try to check screen size with SizerUtil.height and SizerUtil.width, but the value is wrong. i use 15" device

![image](https://user-images.githubusercontent.com/11658522/124715204-ecd6fe80-df2c-11eb-8fa2-46ae86a60422.png)

please help to fix, thanks

Fahmi

**Comment by rehman-00001** [comment]
This code here is the reason for the bug I think:

[util.dart](https://github.com/TechnoUrmish/Sizer/blob/23fb26132c3b2b977635c118764f134e099b5719/lib/util.dart#L30)

```dart
    if (orientation == Orientation.portrait) {
      width = boxConstraints.maxWidth;
      height = boxConstraints.maxHeight;
    } else {
      width = boxConstraints.maxHeight;
      height = boxConstraints.maxWidth;
    }
```

Not sure what was the author's intention behind this.

**Comment by hatch01** [comment]
I made a pull request to fix this : #58
